src/foam/nanos/dig/DUGRule.js

```diff
@@ -182,6 +214,30 @@ foam.CLASS({
     {
       name: 'createdByAgent',
       hidden: true
+    },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.User',
+      name: 'actingUser',
+      writepermissionrequired: true,
+      readpermissionrequired: true,
+      section: 'dugInfo'
+    },
+    {
+      class: 'Boolean',
```

Comment:
Because we set the actingUser to the createdBy on put when the actingUser is null, we need to have this flag to allow skipping the whole actAsUser logic.  I made some changes that removed the use of the flag, so I added the functionality back instead of removing the field.